js/packages/openinference-semantic-conventions/src/trace/SemanticConventions.ts

```diff
@@ -341,6 +341,7 @@ export enum OpenInferenceSpanKind {
   RERANKER = "reranker",
   EMBEDDING = "embedding",
   AGENT = "agent",
+  GUARDRAIL = "guardrail",
```

Comment:
These changes require something called a changeset to be deployed: https://github.com/Arize-ai/openinference/blob/main/js/DEVELOPMENT.md#changesets